### DISCUSSION TITLE
배포 도메인 구입

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#156] [Ideas]
어떤 이름으로 할까요?

![image](https://github.com/user-attachments/assets/4ed4d869-75b8-4e25-bb6b-f9eca7abbdee)

**Comment by clyde-yoon** [comment]
제가 테무에서 하나 구해오겟습니다

**Reply by JiinSeok** [reply]
테무에 도메인 팔아요....?!

**Reply by clyde-yoon** [reply]
싼게 제일좋을거같슴니당

**Comment by JuhyeokC** [comment]
* `albaform.site`
* `albaform.me`

현재 로고 그대로를 제안합니다~
